google/ggrc-core
# GGRC-7641/GGRC-7507 Count of Assessment/Assessment templates is not changed on the Assessment/Assessment template tab after adding new

# Issue description

After creating Assessment, creating or clone Assessment template via Audit summary page, counts in related tabs doesn't update.

# Steps to test the changes

1. Create or open existing audit
2. Click Create assessment button
3. Fill mandatory fields 
4. Click save & add another
5. Expected: Counts of assessments was updated after saving
6. Repeat 1 and 2 steps and click save button
7. Expected: Counts of assessments was updated after loading
8. Return to audit summary page
9. Create assessment template
10. Expected: Counts of assessment templates was updated after loading
11. Repeat with clone assessment template
12. Expected: Counts of assessment templates was updated after loading

# Solution description

Added additional methods for processing this cases

# Sanity checklist




# PR Review checklist
